programs/bpf_loader/src/syscalls/mod.rs

```diff
@@ -18,6 +18,17 @@ use {
         ALT_BN128_PAIRING_ELEMENT_LEN, ALT_BN128_PAIRING_OUTPUT_LEN,
     },
     solana_compute_budget::compute_budget::ComputeBudget,
+    solana_feature_set::bpf_account_data_direct_mapping,
+    solana_feature_set::FeatureSet,
+    solana_feature_set::{
+        self as feature_set, abort_on_invalid_curve, blake3_syscall_enabled,
```

Comment:
rustfmt doesn't seem to care, but can you combine these into one? (not alphabetical, so it'll need to get formatted)
```suggestion
    solana_feature_set::{
        self as feature_set, FeatureSet, bpf_account_data_direct_mapping, abort_on_invalid_curve, blake3_syscall_enabled,
```